### ISSUE TITLE
Term 'element' in 'content' value gets parsed

### ISSUE THREAD

**Issue by patric-eberle** [postcss/autoprefixer#1387]
Version: 10.2.3

If I use the word 'element' in a string value for 'content', the word will be prefixed.

```css
    content: "Element 'div' not allowed as child of element 'span' in this context.";
```

```css
  content: "Attribute width not allowed on -moz-element picture at this point.";
  content: "Attribute width not allowed on element picture at this point.";
```

The same output as input would be expected in this case.

```json
  "browserslist": [
    "> 2%",
    "last 2 years",
    "ie 11",
    "not ie_mob > 0",
    "not dead"
  ]
```

```
├── autoprefixer@10.2.3 
└─┬ stylelint@13.9.0
  └── autoprefixer@9.8.6 
```

**Comment by ai** [comment]
Show me minimum CSS input, current output, and expected output

**Comment by patric-eberle** [comment]
Actually the ones I've posted above:

Input
```css
.foo {
  content: "Element 'div' not allowed as child of element 'span' in this context.";
}
```

Current output
```css
.foo {
  content: "Attribute width not allowed on -moz-element picture at this point.";
  content: "Attribute width not allowed on element picture at this point.";
}
```

Expected output
```css
.foo {
  content: "Element 'div' not allowed as child of element 'span' in this context.";
}
```

**Comment by ai** [comment]
Got it. It is rare to see text in CSS so it was not obvious to me that it was CSS.

Do you want to send PR with a fix (I am busy on another open-source project)? You need to add tests and add the code to `processors.js` to ignore any value processors for the `content` property.

**Comment by ai** [comment]
The fix was released in 10.3.1.
